src/location/util.rs

```diff
@@ -378,6 +842,39 @@ impl LocationUtil {
             //                    .map(|mut row| {})
             //                    .collect()
             //            })?;
+        } else if ob_vsn_major() >= 4
+            && (ob_part_func_type.is_hash_part() || ob_part_func_type.is_key_part())
+        {
+            // parseFirstPartKeyHash
```

Comment:
Make the comment to be a statement rather than a word.

Reply:
Fixed